### PR TITLE
[gms-1283] PaymentSplitter

### PR DESCRIPTION
**Pull request by jasonzwli**

### CODE REVIEW

**Review by Sam-Jeston — COMMENTED** [review]
There are still a few GuildOfGuardians references in the comment blocks

**Comment by jasonzwli** [comment]
> There are still a few GuildOfGuardians references in the comment blocks

oof thought i got them all.

**Review by allan-almeida-imtbl — DISMISSED** [review]

File: test/payment-splitter/MockERC20.sol [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+// Basic ERC20. It is deployed by the tests in order to help testing the GuildOfGuardiansPaymentSplitter ERC20 payment feature
```
Comment (Sam-Jeston) [inline test/payment-splitter/MockERC20.sol:4]: last GoG reference

**Review by ipekt — APPROVED** [review]

**Review by Sam-Jeston — COMMENTED** [review]
1 last GoG reference and lint is failing. Otherwise lgtm

**Review by bruno-imx — APPROVED** [review]

**Review by allan-almeida-imtbl — APPROVED** [review]

**Pull request merged**